### ISSUE TITLE
NTR: Lateral neck mass (Face & neck)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7823]
New term request Lateral neck mass (elthat_130212183249): Including tender lateral cord

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000174
- label: Lateral neck mass
- parents: [Abnormal neck morphology (HP:0025668)](https://hpo.jax.org/app/browse/term/HP:0025668)
- pmid: 
- def: An abnormal enlargement or swelling localized to the side of the neck. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7823
- synonyms:
